src/core/Breadcrumbs/Breadcrumbs.test.tsx

```diff
@@ -131,20 +120,10 @@ it('should restore hidden items when there is enough space again', () => {
 
   expect(container.querySelector('.iui-ellipsis')).toBeFalsy();
   assertBaseElement(container);
-
-  scrollWidthSpy.mockRestore();
-  offsetWidthSpy.mockRestore();
 });
 
 it('should hide first item on very small widths', () => {
-  const scrollWidthSpy = jest
-    .spyOn(HTMLElement.prototype, 'scrollWidth', 'get')
-    .mockReturnValueOnce(250)
-    .mockReturnValueOnce(150)
-    .mockReturnValue(100);
-  const offsetWidthSpy = jest
-    .spyOn(HTMLElement.prototype, 'offsetWidth', 'get')
-    .mockReturnValue(100);
+  jest.spyOn(UseOverflow, 'useOverflow').mockReturnValue([jest.fn(), 1]);
```

Comment:
Kinda does not make sense these tests anymore as there is nothing being mocked/done with width as test says..

Reply:
But it has a special handling when there is only 1 item visible.

Reply:
hmm.. thought test name is kinda wrong then, but minor.. leave it.